pallets/did/src/tests.rs

```diff
@@ -118,7 +119,7 @@ impl Config for Test {
 	type WeightInfo = ();
 }
 
-type DID = Module<Test>;
+type DidModule = Module<Test>;
```

Comment:
The `DID` Module should come out of the `construct_runtime!`-Thingy. THis should go and get replaced with `Did`? Didn't try that tho.